content/de/docs/contribute/participate/_index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Bei SIG Docs mitmachen
+content_type: concept
+weight: 60
+card:
+  name: contribute
+  weight: 60
+---
+
+<!-- overview -->
+
+Die SIG Docs ist eine der 
+[Special Interest Groups (Fachspezifischen Interessengruppen)](https://github.com/kubernetes/community/blob/master/sig-list.md) innerhalb des Kubernetes-Projekts, die sich auf das Schreiben, Aktualisieren und Pflegen der Dokumentation f&uuml;r Kubernetes als Ganzes konzentriert. Weitere Informationen &uuml;ber die SIG findest du unter SIG Docs im [Github Repository der Community](https://github.com/kubernetes/community/tree/master/sig-docs).
+
+SIG Docs begr&uuml;&szlig,t Inhalte und Bewertungen von allen Mitwirkenden. Jeder kann einen
```

Comment:
```suggestion
SIG Docs begr&uuml;&szlig;t Inhalte und Bewertungen von allen Mitwirkenden. Jeder kann einen
```
?
It's also OK to use the actual characters; that's much easier for someone else to read, and Hugo copes just fine with Unicode.